content/k8s/fluxcd/helmrelease/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+content_type: project
+flavours:
+- none
+from_repo: k8s/manual-app-deployment/project-overview
+prerequisites:
+  hard:
+  - k8s/manual-app-deployment/project-overview
+  soft:
+  - k8s/fluxcd/helmrepository
+ready: true
+submission_type: continue_repo
+tags:
+- kubernetes
+- fluxcd
+title: Helmrelease
+---
+
+
+ 
+
+## Helmrelease cert-manager
+
+To load a helmchart from a helmrepository using FluxCD's helmcontroller we
+issue a helmrelease.
+
+To start let's create folders and files for each release
+
+```
+|-- infrastructure
+|   |-- cert-manager
+|   |   |-- kustomization.yaml
+|   |   |-- kustomizeconfig.yaml
+|   |   |-- release.yaml
+|   |   |-- values.yaml
+|   |   |-- namespace.yaml
+|   |   |-- issuer.yaml
+```
+
+```
+# infrastructure/cert-manager/kustomization.yaml
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: cert-manager
+resources:
+  - namespace.yaml
+  - release.yaml
+configMapGenerator:
+  - name: values
+    files:
+      - values.yaml=values.yaml
+configurations:
+  - kustomizeconfig.yaml
+```
+
+```
+# infrastructure/cert-manager/kuztomizeconfig.yaml
+---
+# Kustomize config for enabling HelmRelease values from
+# ConfigMaps and Secrets generated by Kustomize
+nameReference:
+- kind: ConfigMap
+  version: v1
+  fieldSpecs:
+  - path: spec/valuesFrom/name
+    kind: HelmRelease
+```
+
+```
+# infrastructure/cert-manager/release.yaml
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: cert-manager
+  namespace: cert-manager
+spec:
+  releaseName: cert-manager
+  chart:
+    spec:
+      chart: cert-manager
+      sourceRef:
+        kind: HelmRepository
+        name: jetstack
+        namespace: flux-system
+  interval: 24h0m0s
+  install:
+    remediation:
+      retries: 4
+  valuesFrom:
+    - kind: ConfigMap
+      name: values
+```
+
+```
+# infrastructure/cert-manager/namespace.yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cert-manager
+```
+
+```
+# infrastructure/cert-manager/issuer.yaml
+---
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-prod
+  namespace: cert-manager
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: my@email.com
+    privateKeySecretRef:
+      name: letsencrypt-prod
+    solvers:
+      - http01:
+          ingress:
+            ingressClassName: nginx
+```
+
+Now remember to add the folder in the infrastructure kustomization
+
+```
+# infrastructure/kustomization.yaml
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - sources
+  - cert-manager
+```
+
+git push to your main branch and you should notice the containers rollout in cert-manager namespace
+
+```
+kubectl -n cert-manager get pods
+kubectl -n flux-system get helmchart
+```
+
+and before we forget we haven't adde the issuer.yaml yet
```

Comment:
Can you highlight the changed line with a comment or something?

Misspelled `added`.

Or also make the description like:
> And before we forget we haven't added the `issuer.yaml` to the `cert-manager/kustomization.yaml` file: